pkg/apis/tensorflow/v1alpha2/types.go

```diff
@@ -48,6 +48,11 @@ type TFJobSpec struct {
 	// Default to Running.
 	CleanPodPolicy *CleanPodPolicy `json:"cleanPodPolicy,omitempty"`
 
+	// TTL to clean up tf-jobs (temporary before kubernetes adds the
+	// cleanup controller).
+	// Default to infinite.
+	TTLAfterFinished *TFJobTTLAfterFinished `json:"ttlAfterFinished,omitempty"`
```

Comment:
There's discussion about what the right type should be in
https://docs.google.com/document/d/1U6h1DrRJNuQlL2_FYY_FdkQhgtTRn1kEylEOHRoESTc/edit#heading=h.iho4ntmve3bq

I think we should try to follow what they do to be consistent with K8s. They use an int32. I do like your suggestion of using a human readable string. Lets suggest it and see what they say.

Reply:
I use this because I saw something similar in k8s

https://github.com/kubeflow/tf-operator/blob/master/vendor/k8s.io/client-go/tools/clientcmd/helpers.go#L31